styles/styles.css

```diff
@@ -0,0 +1,348 @@
+  left: 13rem;
+  top: 1rem;
+  font-size: 60px;
+  font-family: sans-serif;
+}
+
+header h2 {
+  color: white;
+  position: absolute;
+  font-size: 25px;
+  font-family: sans-serif;
+  font-style: italic;
+  line-height: 36px;
+}
+
+.b-box {
+  width: 30rem;
+  height: 100px;
+}
+
+/*
+.header-img {
+  object-fit: cover;
+  width: 100%;
+  height: inherit;
+  object-position: top;
+}
+*/
+
+.header-img {
+  width: 100%;
+  height: 24rem;
+  background-image: url(../images/header-image.jpg);
+  background-size: cover;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.ftrlogo {
+  height: 30%;
+}
+
+.b2box {
+  height: 70%;
+  width: 40%;
+  margin: 4%;
+  min-width: 227px;
+  max-width: 484px;
+}
+
+.hdrtxt {
+  justify-content: center;
+}
+
+.ckout {
+  margin: 4%;
+}
+
+.art1 {
+  width: 60%;
+  display: inline-block;
+  margin: 10%;
+}
+
+
+
+
+
+
+/* Style the links inside the navigation bar */
+nav {
+  font-family: sans-serif;
+}
+
+nav li {
+  padding: 0 5px 0 5px;
+  position: relative;
+  left: 11rem;
+}
+
+nav li a:after{
+  content: " / ";
+  position: absolute;
+  right: 0;
+  top : 15px;
+}
+
+nav li:last-child a:after{
+    display: none;
+}
+
+nav ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  height: 50px;
+  overflow: hidden;
+  background-color: rgb(235, 235, 235);
+}
+
+nav li {
+  float: left;
+}
+
+nav li a {
+  display: block;
+  color: black;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+/* Change the link color to #111 (black) on hover */
+nav li a:hover {
+  background-color: Grey;
+}
+
+#wrapper {
+  width: 80%;
+  min-height: auto;
+  margin: 2rem auto;
+  overflow: hidden;
+  line-height: 32px;
+  font-size: 16px;
+  font-family: sans-serif;
+}
+
+.flex-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.flex-container > div {
+  background-color: #f1f1f1;
+  margin: 10px;
+  text-align: center;
+  line-height: 75px;
+  font-size: 30px;
+}
+
+.dogimg {
+  width: 100%;
+}
+
+.div1 {
+  width: 98%;
+  display: inline-table;
+}
+
+.div2 {
+  width: 22%;
+  min-width: 275px;
+}
+
+.col {
+    width: 90%;
+    margin: 2rem auto;
+    float: left;
+    padding: 5px;
+}
+
+.row::after {
+    content: "";
+    clear: both;
+    display: table;
+}
+
+.span_3_of_12 {
+  width: 25%;
+}
+
+.span_8_of_12 {
+  width: 66.66%;
+}
+
+.span_9_of_12 {
+  width: 75%;
+}
+
+
+
+.hide {
+  display: none;
+}
+
+.left {
+  float: left;
+  width: 60%;
+  height: auto;
+  text-align: left;
+}
+
+.right {
+  float: right;
+  width: 35%;
+  height: auto;
+  text-align: left;
+}
+
+.left article {
+  padding: 1rem;
+  margin: 1.5rem 0 1rem;
+  box-sizing: border-box;
+}
+
+
+
+#a1 {
+  font-size: 24px;
+  font-family: sans-serif;
+  color: rgb(44, 154, 183);
+}
+
+#a2 {
+  font-size: 24px;
+  font-family: sans-serif;
+  color: rgb(44, 154, 183);
+}
+
+em {
+  font-size: 25px;
+}
+
+
+.body-link {
+  color: rgb(44, 154, 183);

```

Comment:
Be consistent with colors. I'm seeing the css names (hotpink), hex values, and rgb vales. Mxing css names and one other option is fine, but be consistent. I'd steer you to the hex values  like`#e4e4e4` and a few CSS names if you'd like (like hotpink)

RGB's are great for the cases where you need opacity, or things like box shadows

Reply:
updated to hex